flink-runtime/src/main/java/org/apache/flink/streaming/api/functions/sink/legacy/SinkFunction.java

```diff
@@ -32,7 +32,6 @@
  *     org.apache.flink.api.connector.sink2.Sink} interface instead.
  */
 @Internal
-@Deprecated
```

Comment:
why are we changing this tag? 
It is was part of the legacy deprecated API - should this not be deprecated?

Reply:
It should still be deprecated. I remove it since the annotation @Deprecated duplicate with the comments @deprecated. And I follow up the https://github.com/apache/flink/pull/25331